lib/sikuli/facebook.sikuli/facebook.py

```diff
@@ -52,6 +53,12 @@ def wait_for_loaded(self):
         wait(self.fb_logo)
         setAutoWaitTimeout(default_timeout)
 
+    def wait_for_messenger_loaded(self):
+        default_timeout = getAutoWaitTimeout()
+        setAutoWaitTimeout(10)
+        wait(self.messenger_header)
```

Comment:
I am not sure what's the different between
```python
default_timeout = getAutoWaitTimeout()
setAutoWaitTimeout(10)
wait(self.messenger_header)
setAutoWaitTimeout(default_timeout)
```
and 
```python
wait(self.messenger_header, 10)
```

If they are the same, I will prefer use the simple one.